server/container/guice/cassandra-rabbitmq-guice/src/main/java/org/apache/james/CassandraRabbitMQJamesServerMain.java

```diff
@@ -190,6 +190,7 @@ public static GuiceJamesServer createServer(CassandraRabbitMQJamesConfiguration
             .combineWith(BlobStoreCacheModulesChooser.chooseModules(blobStoreConfiguration))
             .combineWith(SearchModuleChooser.chooseModules(searchConfiguration))
             .combineWith(new UsersRepositoryModuleChooser(new CassandraUsersRepositoryModule())
-                .chooseModules(configuration.getUsersRepositoryImplementation()));
+                .chooseModules(configuration.getUsersRepositoryImplementation()))
+            .overrideWith(new DistributedPop3Module());
```

Comment:
Commit title issue: `Bind the Distributed POP3 server within the Distributed POP3 server. `

You mean instead `Bind the Distributed POP3 server within the Distributed server. `?